.github/workflows/validate.yml

```diff
@@ -7,11 +7,14 @@ on:
       - main
       - release-*
   pull_request:
-env:
-  GO_VERSION: 1.22.x
 
 jobs:
   lint:
```

Comment:
I just realized this isn't running in this PR, but it is running in Prow. 🤔

Reply:
Well. 😕 Do you want me to go ahead and delete the file in this PR, then?

Reply:
I guess so, yeah (unless you know a way to re-enable GHA:)

Reply:
Haha, unfortunately I don't have such powers 😆